File: submissions/sm_028_sagar/week_03/day_6/assignment/stringsPlay.js

inp = 'sagar  kadu'
// console.log(removeSpace(inp))
console.log(hypenCase(inp))

function removeSpace(inp){
    let str = ''
    let arr = []
    for(let i = 0; i < inp.length; i++){
        if(inp[i] === ' '){
            arr.push(str)
            str = ''
            continue
        }else{
            str+=inp[i]
        }
    }
    arr.push(str)
    return arr
}


//camelCase
function camelCase(input){
    let arr = removeSpace(input)
    let cnt = 0
    let str = ''
    for(let i = 0; i < arr.length; i++){
        if(arr[i] === ' ' || arr[i] === ''){
            continue
        }else{
            cnt++
            if(cnt == 1){
                str+=arr[i].toLowerCase()
            }
            else{
                str+=arr[i].charAt(0).toUpperCase()+arr[i].slice(1)
            }
        }
    }
    return str
}


function snakeCase(input){
    let arr = removeSpace(input)
    let cnt = 0
    let str = ''
    for(let i = 0; i < arr.length; i++){
        if(arr[i] === ' ' || arr[i] === ''){
            continue
        }else{
            cnt++
            if(cnt === 1){
                str+=arr[i]
            }else{
                str+='_'+arr[i].toLowerCase()
            }
        }
    }
    return str
}


function hypenCase(input){
    let arr = removeSpace(input)
    let cnt = 0
    let str = ''
    for(let i = 0; i < arr.length; i++){
        if(arr[i] === ' ' || arr[i] === ''){
            continue
        }else{
            cnt++
            if(cnt === 1){
                str+=arr[i]
            }else{
                str+='-'+arr[i].toLowerCase()
            }
        }
    }
    return str
}

File: submissions/sm_028_sagar/week_13/day_5/session_1/strings_play.py
input_string = input()

#camelCase = sagarKadu
def camelCase(input):
        arr = input.split()
        string = arr[0] 
        for i in range(1,len(arr)):
                string += arr[i][0].upper() + arr[i][1:]
        print(string)
                
camelCase(input_string)

#snake_case = sagar_kadu
def snake_case(input):
        arr = input.split()
        string = arr[0]
        for i in range(1,len(arr)):
                string += '_'+arr[i]
        print(string)

snake_case(input_string)

#hypen_case = sagar-kadu
def hypen_case(input):
        arr = input.split()
        string = arr[0]
        for i in range(1,len(arr)):
                string += '-'+arr[i]
        print(string)

hypen_case(input_string)